ApiReview/API.Polly.RateLimiting/NoDocs/Polly/RateLimiterResilienceStrategyBuilderExtensions.cs

```diff
@@ -0,0 +1,18 @@
+// Assembly 'Polly.RateLimiting'
+
+using System;
+using System.Threading.RateLimiting;
+using Polly.RateLimiting;
+
+namespace Polly;
+
+public static class RateLimiterResilienceStrategyBuilderExtensions
+{
+    public static ResilienceStrategyBuilder<TResult> AddConcurrencyLimiter<TResult>(this ResilienceStrategyBuilder<TResult> builder, int permitLimit, int queueLimit = 0);
+    public static ResilienceStrategyBuilder<TResult> AddConcurrencyLimiter<TResult>(this ResilienceStrategyBuilder<TResult> builder, ConcurrencyLimiterOptions options);
+    public static ResilienceStrategyBuilder<TResult> AddConcurrencyLimiter<TResult>(this ResilienceStrategyBuilder<TResult> builder, ConcurrencyLimiterOptions options, Action<OnRateLimiterRejectedArguments> onRejected);
```

Comment:
Do we need the `onRejected` parameter overload here when there's a property for it already on `ConcurrencyLimiterOptions`?